bin/asinit

```diff
@@ -6,8 +6,26 @@ const fs = require("fs");
 const path = require("path");
 const colors = require("../cli/util/colors");
 const version = require("../package.json").version;
+const options = require("../cli/util/options");
 
-if (process.argv.length != 3 || process.argv[2] == "--help" || process.argv[2] == "-h") printHelp();
+const asinitOptions = {
+  "help": {
+    "category": "General",
+    "description": "Prints a help message.",
+    "type": "b",
+    "alias": "h"
+  },
+  "yes": {
+    "category": "General",
+    "description": "Skips asking the user if they want to setup an AssemblyScript project.",
+    "type": "b",
+    "alias": "y"
+  }
+};
+
+const cliOptions = options.parse(process.argv, asinitOptions);
```

Comment:
Parse is typically called with `process.argv.slice(2)` so it doesn't attempt to do anything with node or script path

Reply:
That's why. Got it.